client/common/client.c

```diff
@@ -1142,7 +1142,7 @@ BOOL client_common_get_access_token(freerdp* instance, const char* request, char
 		goto cleanup;
 	}
 
-	*token = (const char*)freerdp_utils_aad_get_access_token(log, response, response_length);
+	*token = freerdp_utils_aad_get_access_token(log, (const char*)response, response_length);
```

Comment:
Same here with the token: `freerdp_utils_aad_get_access_token()` returns `char*` and `*token` also has type `char*`, so this cast looks superfluous.

Reply:
I think that comment is for something before a browser refresh?

Reply:
> I think that comment is for something before a browser refresh?

yes, indeed.